api/decorators/Validators.ts

```diff
@@ -93,3 +93,48 @@ class MatchingPasswordsValidator implements ValidatorConstraintInterface {
 export function HasMatchingPasswords(validationOptions?: ValidationOptions) {
   return templatedValidationDecorator(MatchingPasswordsValidator, validationOptions);
 }
+
+@ValidatorConstraint()
+class EventFeedbackValidator implements ValidatorConstraintInterface {
+  validate(feedback: string[]): boolean {
+    return feedback.length <= 3;
+  }
+
+  defaultMessage(): string {
+    return 'No more than 3 event feedback comments can be submitted';
+  }
+}
+
+export function IsValidEventFeedback(validationOptions?: ValidationOptions) {
+  return templatedValidationDecorator(EventFeedbackValidator, validationOptions);
+}
+
+@ValidatorConstraint()
+class FeedbackTypeValidator implements ValidatorConstraintInterface {
+  validate(feedbackType: FeedbackType): boolean {
+    return Object.values(FeedbackType).includes(feedbackType);
+  }
+
+  defaultMessage(): string {
+    return 'Invalid feedback type';
+  }
+}
+
+export function IsValidFeedbackType(validationOptions?: ValidationOptions) {
+  return templatedValidationDecorator(FeedbackTypeValidator, validationOptions);
+}
+
+@ValidatorConstraint()
+class FeedbackStatusValidator implements ValidatorConstraintInterface {
+  validate(feedbackStatus: FeedbackStatus): boolean {
+    return Object.values(FeedbackStatus).includes(feedbackStatus) && feedbackStatus !== FeedbackStatus.SUBMITTED;
+  }
+
+  defaultMessage(): string {
+    return 'Invalid feedback status';
```

Comment:
Some of our error messages (like this) could be more user-friendly. For example, `Invalid feedback status` &rarr; `Feedback status must be one of ['IGNORED', 'ACKNOWLEDGED']`.